lib/absinthe/phase/document/execution/before_resolution.ex

```diff
@@ -1,17 +0,0 @@
-defmodule Absinthe.Phase.Document.Execution.BeforeResolution do
-
-  @moduledoc false
-
-  use Absinthe.Phase
-
-  alias Absinthe.{Blueprint}
-
-  @spec run(Blueprint.t, Keyword.t) :: Phase.result_t
-  def run(%{resolution: resolution} = bp_root, _opts \\ []) do
-    acc = Enum.reduce(bp_root.schema.plugins(), resolution.acc, fn plugin, acc ->
-      plugin.before_resolution(acc)
-    end)
-
-    {:ok, %{bp_root | resolution: %{resolution | acc: acc}}}
-  end
-end
```

Comment:
@bruce note I've decided to go back on my `Before`/`After` resolution phase stuff. It kept producing a lot boilerplate everywhere. Instead i've simply got an option you can pass to the `Resolution` phase that turns on / off running the callbacks.